docs/vue.md

```diff
@@ -20,118 +20,314 @@ To get started, load either the production or development version of Vue in your
 <!-- Production -->
 <script src="//cdn.jsdelivr.net/npm/vue@3/dist/vue.global.prod.js"></script>
 
-<!-- Development (debugging and Vue.js devtools support) -->
+<!-- Development -->
 <script src="//cdn.jsdelivr.net/npm/vue@3/dist/vue.global.js"></script>
 ```
 
-## Basic rendering
+## Template syntax
 
-Docsify will automatically render basic Vue content that does not require `data`, `methods`, or other instance features.
+Vue [template syntax](https://vuejs.org/v2/guide/syntax.html) is used to create dynamic content. With no additinal configuration, this syntax offers several useful features like support for [JavaScript expressions](https://vuejs.org/v2/guide/syntax.html#Using-JavaScript-Expressions) and Vue [directives](https://vuejs.org/v2/guide/syntax.html#Directives) for loops and conditional rendering.
 
 ```markdown
+<!-- Hide in docsify, show elsewhere (e.g. GitHub) -->
+<p v-if="false">Text for GitHub</p>
+
+<!-- Sequenced content (i.e. loop)-->
 <ul>
-  <li v-for="i in 3">{{ i }}</li>
+  <li v-for="i in 3">Item {{ i }}</li>
 </ul>
+
+<!-- JavaScript expressions -->
+<p>2 + 2 = {{ 2 + 2 }}</p>
 ```
 
-The HTML above will render the following:
+<output data-lang="output">
```

Comment:
Why are there new `<output>` elements in the DOM?

Reply:
Nope. New styles applied to a previously unstyled HTML element: `<output>`